plugins/modules/cpanm.py

```diff
@@ -181,6 +189,7 @@ class CPANMinus(ModuleHelper):
         mirror=cmd_runner_fmt.as_opt_val('--mirror'),
         mirror_only=cmd_runner_fmt.as_bool("--mirror-only"),
         installdeps=cmd_runner_fmt.as_bool("--installdeps"),
+        withrecommands=cmd_runner_fmt.as_bool("--with-recommands"),
```

Comment:
```suggestion
        withrecommends=cmd_runner_fmt.as_bool("--with-recommends", "--without-recommends"),
```
This way, if the parameter is:
- `True`, it will pass `--with-recommends`
- `False` it will pass `--without-recommends`
- `None` it will not pass any argument to the command

Reply:
Agreed, did not think about that. So I also used it for install_suggestions.
Strangely, according to https://metacpan.org/dist/App-cpanminus/view/bin/cpanm#-with-recommends,-with-suggests
`--with-recommends` with an `s` at the end and `--without-recommend` without an `s` at the end.
I don't know if it's a typo from in cpanm doc or it's something I did not get with english.

Reply:
Yup I noticed that too, and tested locally, it actually works with both. I suggested _with_ the `s` for consistency.

Reply:
yep, I agree with you.